### DISCUSSION TITLE
Focus browser when clicking a link from another workspace

### DISCUSSION THREAD

**Discussion by victorballester7** [hyprwm/Hyprland#6492] [General]
How would you make the new tab focus in firefox (or any other browser) whenever a link is clicked from another workspace, say for example, from inside a .pdf of a terminal. I mean, I would like to be redirected automatically to the workspace where firefox is located.
Thanks, I appreciate any comment.

**Comment by TKone7** [comment]
Just was looking for the same thing. You can put this in your `hyprland.conf`:
```
misc { 
    ..
    focus_on_activate = true
}
```
[Link to docs](https://wiki.hyprland.org/Configuring/Variables/#misc)
